src/test/java/mate/academy/springbootintro/service/BookServiceTest.java

```diff
@@ -238,38 +250,37 @@ Verify updateBook() method works
     public void updateBook_ValidBookRequestDto_ReturnsBookDto() {
         // Given
         UpdateBookRequestDto updateBookRequestDto = new UpdateBookRequestDto(
-                "Updated Title",
-                "Updated Author",
-                new BigDecimal("19.99"),
-                "updated_cover_image.jpg",
-                "Updated Description"
+                UPDATED_BOOK_TITLE, UPDATED_BOOK_AUTHOR,
+                UPDATED_BOOK_PRICE, UPDATED_BOOK_COVER_IMAGE,
+                UPDATED_BOOK_DESCRIPTION
         );
+
         Book existingBook = new Book();
         existingBook.setId(BOOK_ID);
-        existingBook.setTitle("Original Title");
-        existingBook.setAuthor("Original Author");
-        existingBook.setPrice(new BigDecimal("10.99"));
-        existingBook.setCoverImage("original_cover_image.jpg");
-        existingBook.setDescription("Original Description");
+        existingBook.setTitle(BOOK_TITLE);
+        existingBook.setAuthor(BOOK_AUTHOR);
+        existingBook.setPrice(BOOK_PRICE);
+        existingBook.setCoverImage(BOOK_COVER_IMAGE);
+        existingBook.setDescription(BOOK_DESCRIPTION);

```

Comment:
You should create and use something similar to createBookDto you created in BookControllerTest - in all places:)